alts/src/test/java/io/grpc/alts/internal/AltsAuthContextTest.java

```diff
@@ -54,11 +60,14 @@ public void setUp() {
                     .setMinor(TEST_MIN_RPC_VERSION_MINOR)
                     .build())
             .build();
+    Identity.Builder peerIdentity = Identity.newBuilder()
+        .setServiceAccount(TEST_PEER_SERVICE_ACCOUNT);
+    peerIdentity.putAllAttributes(TEST_PEER_ATTRIBUTES);
```

Comment:
If we define the map to be a local variable, it doesn't need to be capitalized. We usually do that for some static strings. We can also add some fake values and see if what we got equals what we expect. The key and value of that fake value could be capitalized as a static string.

Reply:
Added thanks!